print-802_15_4.c

```diff
@@ -230,3 +230,18 @@ ieee802_15_4_if_print(netdissect_options *ndo,
 	ndo->ndo_protocol = "802.15.4_if";
 	return ieee802_15_4_print(ndo, p, h->caplen);
 }
+
+/* For DLT_IEEE802_15_4_TAP*/
+u_int
+ieee802_15_4_tap_if_print(netdissect_options *ndo,
+                      const struct pcap_pkthdr *h, const u_char *p)
+{
+	uint8_t version = EXTRACT_U_1(p);
```

Comment:
This needs to check whether `h->caplen` is >= 4; if it's not, you're fetching past the end of the packet buffer.